amps/ags/rm-automation/rm-automation-community-rest-api/src/test/resources/testng.xml

```diff
@@ -1,9 +1,16 @@
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 
 <suite name="TestNG AllTestSuite" configfailurepolicy="continue" verbose="1" time-out="300000">
+   <test name="restapi - test priorities" >
```

Comment:
This fix for the AGS tests not working due to re-ordering after the switch from Travis to GHA is mostly fine. However I'd still prefer for the tests to be idempotent and not depend on the ordering if possible. It would be nice to figure out exactly what breaks these tests and if it's possible to make them independent from ordering.